### PR TITLE
[Feature] Make homepage look normal

### PR DESCRIPTION
**Pull request by etainad008**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+const Homepage = () => {
+    return (
+        <>
+            <MainContent></MainContent>
+        </>
+    );
 }
 
-export default App;
+export default Homepage;
```
Comment (idandev) [inline src/App.tsx:44]: Add newline

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+  background-color: var(--accent-red);
+}
+
+#b2 {
+  background-color: var(--accent-green);
+}
+
+.img-fluid {
+  border-radius: 0.75rem;
+}
```
Comment (idandev) [inline src/index.css:46]: Use bootstrap instead

**Review by idandev — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,41 +1,45 @@
 import React, { useState } from 'react';
-import './styles.css';
-import 'bootstrap/dist/css/bootstrap.min.css';
+import './styles.css'
```
Comment (martinalebachew) [inline src/App.tsx:2]: Please add semicolons

Comment (idandev) [reply src/App.tsx]: FIxed

**Review by idandev — APPROVED** [review]
LGTM

**Pull request merged**